--- a/src/sockmap.c	2018-07-03 21:56:33.000000000 +0000
+++ b/src/sockmap.c	2018-08-13 13:28:22.584777087 +0000
@@ -8,10 +8,11 @@
 
 #include "proxy.h"
 
+#define _FILE_OFFSET_BITS 64
 #ifdef WITHSPLICE
 
 #include <fcntl.h>
-ssize_t splice(int fd_in, loff_t *off_in, int fd_out, loff_t *off_out, size_t len, unsigned int flags);
+ssize_t splice(int fd_in, off_t *off_in, int fd_out, off_t *off_out, size_t len, unsigned int flags);
 #ifndef SPLICE_F_MOVE
 #define SPLICE_F_MOVE           0x01
 #endif
